WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -70,6 +72,9 @@ private extension ManualTrackingViewController {
         configureTitle()
         configureDismissButton()
         configureAddButton()
+        // Disables the ability to dismiss the view controller via a pull-down gesture, in order to avoid losing unsaved changes.
+        isModalInPresentation = false
```

Comment:
Is this really necessary? I see that in other places in our app where we have the same use case we do not have to set this property

Reply:
It is not necessary indeed 👍